backstop_data/engine_scripts/cookies.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "domain": ".www.yourdomain.com",
```

Comment:
Is this file supposed to be customized for PatternFly?

Reply:
Not this file, they provide a lot of example/templates for different configs/scripts you might want to use when testing. This would set a cookie with some value you might need for chromium to access a test URL